core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -830,6 +848,43 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
 
   }
 
+  it should "ack pubsub message if message is double delivered" in withTestDataApiServices { services =>
+    val timeout = 30000 milliseconds
+    val interval = 250 milliseconds
+    val importUuid = UUID.randomUUID()
+
+    setUpMockImportServiceToReturnStatus(services, ImportStatuses.Done)
```

Comment:
What I really wanted to do here was inline `setUpMockImportServiceToReturnStatus` (as well as the `setUpMockImportServiceToThrowAnError` in the other two tests) but the compiler wouldn't let me use the mock DSL from within this scope (it complained about me trying to use `when`).  I'm not sure I understand why, but open to advice if doing so is reasonable.

Reply:
Fixed in 0bd38c139e83f5050c6c471b75df87313bcc6572